lib/cesium_link/links.ex

```diff
@@ -64,7 +64,11 @@ defmodule CesiumLink.Links do
 
   """
   def list_unarchived_links_by_index do
-    Repo.all(from l in Link, where: l.archived == false, order_by: [asc: l.index])
+    Link
+    |> where([l], fragment("? <= now() OR ? IS NULL", l.publish_at, l.publish_at))
+    |> where([l], l.archived == false)
```

Comment:
```suggestion
    |> where([l], not l.archived)
```

Reply:
Confirm it is working with a test because I believe it does not work the same in the Ecto queries :bulb: